lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2625,6 +2740,11 @@ SwiftLanguageRuntime::GetRuntimeUnwindPlan(ProcessSP process_sp,
                                            RegisterContext *regctx,
                                            bool &behaves_like_zeroth_frame) {
   LLDB_SCOPED_TIMER();
+  auto log_expected = [](llvm::Error error) {
+    Log *log = GetLog(LLDBLog::Unwind);
+    LLDB_LOG_ERROR(log, std::move(error), "{0}");
```

Comment:
@felipepiovezan The error and format string are in reverse order.

Reply:
oh wow, good catch... I had forced all error messages to reproduce, but  it was before this helper function was introduced :(

Reply:
Wait, are you sure?  There are a bunch of examples in the code base in this same order:
```
lldb/source/API/SBTarget.cpp
1164:      LLDB_LOG_ERROR(GetLog(LLDBLog::Breakpoints), expected_vector.takeError(),
1165-                     "invalid breakpoint name: {0}");
1166-      return false;

lldb/source/API/SBStream.cpp
103:    LLDB_LOG_ERROR(GetLog(LLDBLog::API), file.takeError(),
104-                   "Cannot open {1}: {0}", path);
105-    return;

lldb/source/Breakpoint/Watchpoint.cpp
44:      LLDB_LOG_ERROR(GetLog(LLDBLog::Watchpoints), std::move(err),
45-                     "Failed to set type: {0}");
46-    } else {
--
57:        LLDB_LOG_ERROR(GetLog(LLDBLog::Watchpoints), std::move(err),
58-                       "Failed to set type: Typesystem is no longer live: {0}");
59-    }

lldb/source/Symbol/Type.cpp
644:        LLDB_LOG_ERROR(
645-            GetLog(LLDBLog::Symbols), std::move(err),
646-            "Unable to construct void type from TypeSystemClang: {0}");
```

Reply:
false alarm. I overlooked that this is `LLDB_LOG_ERROR` which I was not aware of.